bin/default_pythons

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
 DEFAULT_PYTHON_VERSION="python-3.6.8"
+
+if [[ $PLOTLY_IS_AIRGAPPED -eq 1 ]]; then
```

Comment:
Can you quote the variable and use `== "1"` for the comparison? Since it gets passed in as an environment variable, it's not _really_ an integer...

```shell
if [[ "$PLOTLY_IS_AIRGAPPED" == "1" ]]; then
```